README.md

```diff
@@ -48,7 +49,9 @@ $ docker run --name pdns \
   * `MYSQL_USER=root`
   * `MYSQL_PASS=root`
   * `MYSQL_DB=pdns`
+  * `MYSQL_DNSSEC=yes`
 * Want to disable mysql initialization? Use `MYSQL_AUTOCONF=false`
+* DNSSEC is disabled by default
```

Comment:
=> DNSSEC is disabled by default, to enable _<describe whats necessary to use it>_

Reply:
Done